urdf/include/urdf/model.h

```diff
@@ -44,6 +44,7 @@
 #include <tinyxml.h>
 #include <boost/shared_ptr.hpp>
 #include <boost/weak_ptr.hpp>
+#include <ros/ros.h>
```

Comment:
@wjwwood correct me if I'm wrong, but I think the package.xml needs `<build_export_depend>roscpp</build_export_depend>` because of this addition